design/commands.md

```diff
@@ -103,9 +103,9 @@ $ sd_cmd exec foo/bar@1 -baz sample
  - **binary** - A script or binary that can be downloaded and directly executed.
 
 ```bash
-$ curl -o /opt/sd/commands/foo/bar/1.0.1 https://store/v1/commands/foo/bar/1.0.1
-$ chmod +x /opt/sd/commands/foo/bar/1.0.1
-$ /opt/sd/commands/foo/bar/1.0.1 -baz sample
+$ curl -o /opt/sd/commands/foo/bar/1.0.1/foobar.sh https://store/v1/commands/foo/bar/1.0.1
```

Comment:
This way, are we suggesting that there can be multiple `.sh` files under `/foo/bar/`?

Reply:
I think there will be no multiple `.sh` files under `/foo/bar`, because the `.sh` files will be under `/foo/bar/<version>`. Also the `.sh` file and version is one-to-one relationship, so there will be one `.sh` file under `/foo/bar/<version>`.

Reply:
@yuichi10 Cool, thanks for the explanation